src/sentry/interfaces/sdk.py

```diff
@@ -49,12 +49,7 @@ class Sdk(Interface):
     @classmethod
     def to_python(cls, data):
         name = data.get('name')
-        if not name:
-            raise InterfaceValidationError("No 'name' value")
-
         version = data.get('version')
-        if not version:
```

Comment:
Would an SDK not send its version because of PII stripping?

Reply:
yes exactly, relay might strip this value out